upstream-community-operators/kubeflow/1.2.0/kubeflow.v1.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,287 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - nonResourceURLs:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: kubeflow-operator
+      deployments:
+      - name: kubeflow-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: kubeflow-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: kubeflow-operator
+            spec:
+              containers:
+              - command:
+                - kfctl
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: kubeflow-operator
+                image: aipipeline/kubeflow-operator:v1.2.0
+                imagePullPolicy: Always
+                name: kubeflow-operator
+                resources: {}
+              serviceAccountName: kubeflow-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  maturity: alpha
+  links:
+  - name: Kubeflow
+    url: https://www.kubeflow.org/
+  provider:
+    name: Kubeflow
+  maintainers:
+    - name: Animesh Singh
+      email: singhan@us.ibm.com
+    - name: Tommy Li
+      email: tommy.chaoping.li@ibm.com
+    - name: Weiqiang Zhuang
+      email: wzhuang@us.ibm.com
+    - name: Evan Hataishi
+      email: evan.hataishi@ibm.com
+  keywords: 
+    - Kubeflow
+    - Operator
+    - IBMCloud
+    - GCP
+    - OpenShift
+  version: 1.2.0
+  replaces: kubeflow.v1.1.0
```

Comment:
Don't think we have v1.1.0 version.

Reply:
Ah, right. This repository was never updated with the 1.1.0 operator. Fixed the replaces for now.